build.bat

```diff
@@ -1,9 +1,15 @@
-set /P FiltaQuillaRev=<revision.txt

```

Comment:
Sorry but I need to build in windows, cannot (and don't want to) just retire my batch job - also the fact that I keep backups of each build has saved my ass more than one time. Using the revision.txt to autoincrement each time I build.

Reply:
Np. I'll leave it as it was. It just needs some attention IMHO.

Reply:
Reverted original script.